### PR TITLE
ro_RO Fix style, diacritics; improve translations; translate more strings

### PR DESCRIPTION
**Pull request by alexei**

### CODE REVIEW

**Comment by alexei** [comment]
@jbarrett if you have any feedback, please shoot. I'll take another look over all translations when I get the chance.

File: locales/ro_RO/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -97,19 +99,21 @@ msgstr "%sClick dreapta%s în bara de căutare"
...
 msgstr "Dublu click pe %sduckduckgo.safariextz%s"
 
 #. https://duckduckgo.com/params in the end under " Source:"
+#, fuzzy
 msgid "A string to identify the source."
 msgstr "Un șir pentru a identifica sursa."
 
+#, fuzzy
 msgctxt "settings"
 msgid "ATB Dismiss"
```
Comment (alexei) [inline locales/ro_RO/LC_MESSAGES/duckduckgo.po]: What's an ATB?

File: locales/ro_RO/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -147,49 +151,51 @@ msgstr "Reclama este nepotrivită"
...
 msgid "Add DuckDuckGo to %s"
-msgstr "Adăugați DuckDuckGo la %s"
+msgstr "Adaugă DuckDuckGo la %s"
 
 #. https://duckduckgo.com/search_box near the top
-#. e.g.: Add a DuckDuckGo search box to your site! 
+#. e.g.: Add a DuckDuckGo search box to your site!
 msgid "Add a %s search box to your site!"
-msgstr "Adăugați o căsuță de căutare %s la site-ul dumneavoastră."
+msgstr "Adaugă o căsuță de căutare %s pe site-ul tău."
```
Comment (alexei) [inline locales/ro_RO/LC_MESSAGES/duckduckgo.po]: Should end with a `!`

File: locales/ro_RO/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (epsimatt) [reply locales/ro_RO/LC_MESSAGES/duckduckgo.po]: @alexei I think it's about the anonymous A/B test they have been doing as a way to improve DuckDuckGo (reference: https://duck.co/help/privacy/atb)

Comment (alexei) [reply locales/ro_RO/LC_MESSAGES/duckduckgo.po]: @epsimatt yeah, I looked it up on the Internet and found there was an `atb` query param used for A/B tests. I wonder now what does it mean by `ATB Dismiss` 'cause I'm not sure how to translate that

Comment (jbarrett) [reply locales/ro_RO/LC_MESSAGES/duckduckgo.po]: This one needs a little background.

These tokens are extracted from data which describe user settings. Usually these strings are presented directly on settings pages. I think this "setting" is to note that you have dismissed the prompt to install the DDG browser extension, but it doesn't make sense to make it a configurable setting to re-enable this prompt.

So, the setting needs a string to describe it, but a couple of those strings do not make it to presentation. A note is required for this on the pot file.

In short, I think you are free to ignore this one.

**Comment by jbarrett** [comment]
Many thanks @alexei!

Do you wish to resolve your review comment now or can this change wait? What you've added is a major improvement either way.

**Comment by alexei** [comment]
I'm done for now. I'll be watching the repo and update the ro_RO translation if I can 👍

**Comment by jbarrett** [comment]
Super - many thanks @alexei!
